src/content/Team.yaml

```diff
@@ -1,27 +1,24 @@
 header:
-    title: Über uns
+    title: Chi siamo
     content: |-
-      <p>Kurze Philosophie</p>
-      Blindtext wir sind wahnsinnig super, richtig reich und schön
-      wir lieben euch. Virus schleich dich....
-      Blindtext.
+      <p>la nostra philosophia</p>
+      Un progetto que collequa professionisti situati in Austria tra Sardegne 
 members:
   - name: Elke Oberleitner
     id: elke
-    kurzbio: |-
-      Studium Kommunikationswissenschaften und Philosophie.
-      Univ. Salzburg / AT
-      Zertifizierte DAF/DAZ Trainerin & ÖIF Zertifikation
+    Bio: |-
```

Comment:
changing the tags like kurzbio to "Bio" won't work. only changing the content. you will have to revert that.